libraries/AP_Param/AP_Param.cpp

```diff
@@ -1823,25 +1846,33 @@ void AP_Param::set_float(float value, enum ap_var_type var_type)
 /*
   parse a parameter file line
  */
-bool AP_Param::parse_param_line(char *line, char **vname, float &value)
+bool AP_Param::parse_param_line(char *line, char **vname, float &value, bool &read_only)
 {
     if (line[0] == '#') {
         return false;
     }
     char *saveptr = nullptr;
-    char *pname = strtok_r(line, ", =\t", &saveptr);
+    char *pname = strtok_r(line, ", =\t\r\n", &saveptr);
```

Comment:
A comment explaining the add of these might be a good idea. It took me a bit to understand why you were adding them as delimiters.

Reply:
done